### ISSUE TITLE
denoise profile, preset "luma (use on 2nd instance)"

### ISSUE THREAD

**Issue by AxelG-DE** [darktable-org/darktable#2633]
I am not hundred percent sure, this is a bug, however, 
- when I use `denoise profile, preset "luma (use on 2nd instance)` (indeed I have chroma on 1st instance=on)
- the standard mode is "wavelets". 
- For finer details in 2.6.2 I sometimes switched to "non-local-means". 

Since some time in dt-2.7, when I do this, there is **no more visible difference** between those two.

The former behavior, where "wavelets" and "non-local-means" created diff. results, was more logic to me.

**Comment by rawfiner** [comment]
You mean that switching to nlmeans has no effect after applying the luma preset
Have you tried with opacity set to 100% instead of 70%?
After changing to nlmeans, does changing nlmeans parameters impact the result?

**Comment by AxelG-DE** [comment]
Dear @rawfiner 
>You mean that switching to nlmeans has no effect after applying the luma preset?

almost no effect, yes, where in the past the effect was pretty noticeable.

> Have you tried with opacity set to 100% instead of 70%?

yes. indeed the opacity has some effect but switching between wavelets and nlmeans almost no, as well

> After changing to nlmeans, does changing nlmeans parameters impact the result?

The parameters in nlmeans do work, so touching them does change something. Can it be, they have got recently some different defaults? 
![Screenshot_20190530_120426small](https://user-images.githubusercontent.com/49004103/58625701-4642bd00-82d3-11e9-8084-dd9c641efc0b.png)

**Comment by rawfiner** [comment]
Ok thanks
No the defaut parameters haven't changed
Have you tried on picture with various noise levels?
Could you share a raw with me, along with its xmp, so that I can test with the same image?

**Comment by rawfiner** [comment]
Also, did you zoom at 100%?

**Comment by AxelG-DE** [comment]
Dear @rawfiner indeed I check with 100% :-)

I could provide a sample later, yes. Does that request mean, currently you cannot reproduce?

If I want to also provide sections before and after plus wavelet / nlmeans, I'd need some time, as currently other prios keep me away from PC...

RAW directly here, with its 24MB? I don't have a cloud... 
Or maybe I find it already in one of my pixels.us playraw edits :-)

**Comment by rawfiner** [comment]
Ok, no need for the raw, I just tested, and I can reproduce. The differences are indeed very subtle.
It is not an issue though, it is just some kind of coincidence I think.
I don't think there is anything to do about this, especially since wavelets are faster to compute than nlmeans, so better stick with wavelets. :-)
What do you think?

**Comment by AxelG-DE** [comment]
What moves me is the memory, that recently in the same mode the difference was not that subtile.

Theoretically that could also mean that the wavelet curve was buggy in the past and now works correctly. I would need to go back to 2.6.2 and try out and also reset that curve (which I thought, I did) ;-) 

In that past time of 2.6.2 (and earlier master IIRC) sometimes wavelet was (naturally) too blodgy and hence I switched that second instance to nlmeans and details increased. As I always look for max details, when I denoise, I tried that way again and found oooops suddenly almost no effect, that's how it came up....
